apps/payments/next/middleware.ts

```diff
@@ -20,6 +20,7 @@ export function middleware(request: NextRequest) {
   const PAYPAL_SCRIPT_URL = 'https://www.paypal.com';
   const PAYPAL_API_URL = process.env.CSP__PAYPAL_API;
   const PAYPAL_OBJECTS = 'https://www.paypalobjects.com';
+  const PROFILE_CLIENT_URL = process.env.PROFILE_CLIENT_CONFIG__URL;
```

Comment:
Looks like a different URL is used depending on whether the MzA account uses the default image or has an uploaded profile image. Could you please add support for both?

Reply:
I've added line `PROFILE_SERVER__IMG_URL=http://localhost:1112` to the .env file. It isn't a particularly well-documented route definition (i.e. I have no idea what it is in stage or prod), so we'll want to follow up to make sure it gets added when we deploy this

Reply:
Thanks to @xlisachan for these links:
> https://github.com/mozilla-it/webservices-infra/blob/main/fxa/k8s/fxa/values-stage.yaml#L184
https://github.com/mozilla-it/webservices-infra/blob/main/fxa/k8s/fxa/values-prod.yaml#L268

@StaberindeZA do you know what the correct course of action is around this to make sure it gets linked correctly?

Reply:
Do you mean to get the correct URLs for Stage? Probably best to check with FxA or see what the values are for `fxa-settings` if they have similar CSP rules.

Just as an FYI, currently `payments-next` isn't deployed to Prod, so no need to provide those details yet.

Reply:
Yep, just wanted to make sure I wasn't missing any steps around adding/changing .env files. Sounds like we'll just do a round of it all when we're deploying to prod, so no need to be concerned yet.